tests/unit-tests/dds/DCPS/RTPS/RtpsCoreTypeSupportImpl.cpp

```diff
@@ -64,3 +114,31 @@ TEST(RtpsCoreTypeSupportImpl, ExtractParameter)
   extract_Parameter_user_data("");
   extract_Parameter_user_data("extract_Parameter_user_data");
 }
+
+// ---------- ---------- ---------- ---------- ----------
+TEST(RtpsCoreTypeSupportImpl, InsertFilterResult_t)
+{
+  std::vector<CORBA::Long> v;
```

Comment:
I think having `vector` involved is unnecessary complication.  Sequences are just like vectors, so just use the sequence itself instead of first using a vector and then copying each element into a corresponding sequence.

Reply:
I was trying to use vector as a generic way to pass elements to a generic sequence creation function. Since std::vector does not need explicit setting length as sequence does, we can easily test any length of sequences (including 0) using vector as input source for insertion, or as comparison for exertion. They are all changed to const references as suggested.